.travis.fedora.sh

```diff
@@ -11,15 +11,26 @@ make package
 # /proc and /sys in the chroot).
 mkdir -p /root/rpmbuild/SOURCES
 cp package/* /root/rpmbuild/SOURCES
-rpmbuild -bb -D "fedora_version 25" -D "jobs `nproc`" package/*.spec
+rpmbuild -bb --with coverage -D "fedora_version 25" -D "jobs `nproc`" package/*.spec
```

Comment:
No we use Fedora 31 in the Docker image, I guess we should update also this define...